Gol 61 creating route

Olegsuus opened the pull request.

Olegsuus committed:
GOL-62:Сделал вывод баннеров и юзеров через пагинацию

Olegsuus committed:
GOL-62:Сделал вывод баннеров и юзеров через пагинацию

RyabininAleck commented on internal/db/storage.go:
```diff
@@ -273,6 +273,75 @@ func (db *DataBase) AddAccessLevel(level *models.AccessLevel) error {
+	var banners []models.Banner
+	offset := (page - 1) * size
+
+	query := "SELECT id, title, text, url, created_at, updated_at, owner_id, f_id FROM Banners LIMIT $1 OFFSET $2"
```
Раздели запрос на строки. Так удобнее читать

Olegsuus committed:
GOL-63:
1)Написал функцию для вывода баннеров с их tags
2)почистил код от лишних комментариев

Olegsuus committed:
GOL-63:
1)добавил метод для обновления данных юзера

Olegsuus committed:
GOL-67:
Написал тесты для функций в storage

Olegsuus committed:
GOL-67:
test commit
